src/v1.js

```diff
@@ -556,3 +556,34 @@ const triggerHookCommon = async function({req, res, hook, payload, firedBy}) {
   }
 };
 
+/** 
+ * Get lastfires for a given hookGroup and hookId
+*/
+builder.declare({
+  method:       'get',
+  route:        '/hooks/:hookGroupId/:hookId/listLastFires',
+  name:         'listLastFires',
+  idempotent:   true,
+  output:       'list-lastFires-response.yml',
+  title:        'list lastFires for a given hookId',
+  stability:    'experimental',
+  description: [
+    'This endpoint will return a list of all lastfire definitions for the given `hookGroupId`',
+    'and hookId.',
```

Comment:
Let's re-word this from the user perspective: "This endpoint will return information about the the last few times this hook has been fired, including whether the hook was fired successfully or not."